apps/main-cf.yml.j2

```
@@ -180,7 +188,11 @@ Resources:
     Type: AWS::CloudFormation::Stack
     Properties:
       Parameters:
-        JobQueueArn: !GetAtt Cluster.Outputs.JobQueueArn
+        SharedJobQueueArn: !GetAtt Cluster.Outputs.SharedJobQueueArn
+        {% for job_type, job_spec in job_types.items() if 'Shared' not in job_spec['compute_environment']['name'] %}
```

Any reason not to use a more strict comparison like this?
```suggestion
        {% for job_type, job_spec in job_types.items() if job_spec['compute_environment']['name'] != 'Shared' %}
```

---

There are several places where this change could be made.